docs/releasing.md

```diff
@@ -3,6 +3,14 @@
 <!-- SPDX-License-Identifier: CC0-1.0 -->
 <!-- SPDX-FileCopyrightText: 2021-2024 The Foundation for Public Code <info@publiccode.net>, https://standard.publiccode.net/AUTHORS -->
 
+This step-by-step guide walks through the process of preparing a release for Standard for Public Code, from reviewing the current state of the `develop` branch to sending the final files for print.
+It should cover everything needed ensure the release is properly updated, reviewed, and tagged, including updating relevant documentation, conducting reviews, and publishing the release.
```

Comment:
This is a complicated sentence - it's hard to parse at first read whether this is a long list, or whether it contains a subclause.

```suggestion
It should cover everything needed ensure the release is properly updated, reviewed, and tagged - including updating relevant documentation, conducting reviews, and publishing the release.
```

Reply:
In the spirit of simple language, I'd suggest we break this into two sentences.

Perhaps:

> It should cover everything needed ensure the release is properly updated, reviewed, and tagged.
> This includes updating relevant documentation, conducting reviews, and publishing the release.

Reply:
That's the best solution yet. Is it necessary to mentions reviews in both sentences?

Reply:
ah, indeed, no need to mention reviews twice, agreed

Thank you.

I'll adjust the text and re-push.